stdppx/dune

```diff
@@ -2,7 +2,6 @@
  (name stdppx)
  (public_name ppxlib.stdppx)
  (libraries
-  ocaml-compiler-libs.shadow
   sexplib0
   stdlib-shims)
- (flags (:standard -open Ocaml_shadow -safe-string)))
```

Comment:
same here about `-open Ocaml_shadow`

Reply:
I've also put this one back. Now, that I have `(implicit_transitive_deps false)`, do I still need it?

Reply:
Nope, you shouldn't need it.